### DISCUSSION TITLE
Add Typescript support

### DISCUSSION THREAD

**Discussion by code-matter** [tremorlabs/tremor-npm#124] [Ideas]
I think typescript would be beneficial for Tremor. Especially for the data types

**Comment by mitrotasios** [comment]
Hey @code-matter. The library does fully support TypeScript. Are you facing any issues in that regard?

**Reply by code-matter** [reply]
> Hey @code-matter. The library does fully support TypeScript. Are you facing any issues in that regard?

Literally fixed my problems the second I posted this. Sorry for that!
